modules/dnn/CMakeLists.txt

```diff
@@ -83,6 +86,10 @@ else()
   set(sources_options EXCLUDE_OPENCL)
 endif()
 
+if(HAVE_TENGINE)
+  list(APPEND include_dirs ${TENGINE_INCLUDE_DIRS})
+  list(APPEND libs ${TENGINE_LIBRARIES})
```

Comment:
> No rule to make target `lib/libtengine.a`

In case of "build" mode CMake should get here the target name instead of library path. It is necessary for creating dependencies.

Reply:
done